docs/content/preview/yugabyte-platform/back-up-restore-universes/configure-backup-storage.md

```diff
@@ -15,62 +15,76 @@ Depending on your environment, you can save your YugabyteDB universe data to a v
   ![img](/images/yp/backup-aws-access-control.png)
 - Create Bucket policy to enable access to the objects stored in the bucket.
 
-## Network File Share
+## Network File System
+
+You can configure Network File System (NFS) as your backup target, as follows:
+
+1. Navigate to **Configs > Backup > Network File System**.
 
-You can configure Network File Share (NFS) as your backup target as follows:
+2. Click **Create NFS Backup** to access the configuration form shown in the following illustration:<br><br><br>
 
-1. Navigate to **Configs** > **Backup** > **NFS**.
-3. Complete the **NFS Storage Path** field by entering `/backup` or another directory that provides read, write, and access permissions to the SSH user of the YugabyteDB Anywhere instance.
-3. Click **Save**.
+   ![NFS Cloud Provider Configuration](/images/yp/cloud-provider-configuration-backup-nfs.png)<br><br>
 
-![NFS Cloud Provider Configuration](/images/yp/cloud-provider-configuration-backup-nfs.png)
+3. Use the **Configuration Name** field to provide a meaning name for you backup configuration.
+
+4. Complete the **NFS Storage Path** field by entering `/backup` or another directory that provides read, write, and access permissions to the SSH user of the YugabyteDB Anywhere instance.
+
+5. Click **Save**.
 
 ## Google Cloud Storage
 
-You can configure Google Cloud Storage (GCS) as your backup target as follows:
+You can configure Google Cloud Storage (GCS) as your backup target, as follows:
 
-1. Navigate to **Configs** > **Backup** > **GCS**.
+1. Navigate to **Configs > Backup > Google Cloud Storage**.
 
-3. Complete  **GCS Bucket** and **GCS Credentials** fields.
+2. Click **Create GCS Backup** to access the configuration form shown in the following illustration:<br><br><br>
 
-   For information on how to obtain GCS credentials, see [Cloud Storage authentication](https://cloud.google.com/storage/docs/authentication).
+   ![GCS Cloud Provider Configuration](/images/yp/cloud-provider-configuration-backup-gcs.png)<br><br>
 
-4. Click **Save**.
+3. Use the **Configuration Name** field to provide a meaning name for you backup configuration.
+
+4. Complete  **GCS Bucket** and **GCS Credentials** fields.
+
+   For information on how to obtain GCS credentials, see [Cloud Storage authentication](https://cloud.google.com/storage/docs/authentication).
 
-![GCS Backup](/images/yp/cloud-provider-configuration-backup-gcs.png)
+5. Click **Save**.
 
 You can configure access control for the S3 bucket as follows:
 
 - Provide the required access control list (ACL) and set it as either uniform or fine-grained (for object-level access).
 - Add permissions, such as roles and members.
 
-## Microsoft Azure
+## Azure Storage
 
-You can configure Azure as your backup target as follows:
+You can configure Azure as your backup target, as follows:
 
-1. Create a storage account in Azure as follows:
+1. Create a storage account in Azure, as follows:
```

Comment:
Why is Azure the only one with instructions for creating the storage account?

Reply:
The other sections have links to the provider docs. 
I guess whoever wrote the original doc had a reason for adding the content instead of a link. I'll leave it as is for now.